AutocompleteClientTests/FW/Logic/Request/QuizzesQueryRequestBuilderTests.swift

```diff
@@ -23,28 +24,30 @@ class QuizzesQueryRequestBuilderTests: XCTestCase {
     }
 
     func testQuizQueryBuilder_NextQuestion() {
-        let query = CIOQuizQuery(quizId: self.quizId, answers: self.answers, versionId: self.versionId)
+        let query = CIOQuizQuery(quizId: self.quizId, answers: self.answers, versionId: self.versionId, sessionId: self.sessionId)
         builder.build(trackData: query)
         let request = builder.getQuizRequest(finalize: false)
```

Comment:
Should we want to change the `finalize` parameter here to something else? I think it's okay to leave it as is if we don't think it's necessary to change

Reply:
Yea I was thinking of changing to like results or getResults. But I found finalize to be the simplest? Not sure if you have any suggestions.